versions/development/SPEC.md

```diff
@@ -78,7 +78,7 @@
       * [Struct Member Access](#struct-member-access)
       * [Importing Structs](#importing-structs)
 * [Namespaces](#namespaces)
-* [Scope](#scope)
+* [Value Scopes and Reachability](#value-scopes-and-reachability)
```

Comment:
I'm not used to seeing the term `Reachability` in this context. Most language descriptions I've seen use `Visibility` to describe this concept.

Reply:
I'll replace with visibility